HIP-Basic/moving_average/main.hip

```diff
@@ -101,7 +101,7 @@ int main()
     std::transform(h_input.begin(),
                    h_input.end(),
                    h_input.begin(),
-                   [](unsigned int i) { return i % window_size; });
+                   [&](unsigned int i) { return i % window_size; });
```

Comment:
Better to see the error reported in the description first. Also, better to have fixes for different types of errors in different PRs.